src/core/thole.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright (C) 2010,2011,2012,2013,2014,2015,2016 The ESPResSo project
+  Copyright (C) 2002,2003,2004,2005,2006,2007,2008,2009,2010 
+    Max-Planck-Institute for Polymer Research, Theory Group
+  
+  This file is part of ESPResSo.
+  
+  ESPResSo is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  
+  ESPResSo is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+*/
+
+#include "utils.hpp"
+#include "thole.hpp"
+
+#ifdef THOLE
+#include "communication.hpp"
+
+bool thole_active = false;
```

Comment:
You're right, but then it is more cheap to simply check `thole_s != 0 && q1q2 != 0` in thole.hpp every time I think. This originally was to avoid `check_for_bond_between_particles`.